apps/api/src/bootstrap.ts

```diff
@@ -52,12 +51,7 @@ export async function bootstrap(
     };
   }
 
-  let app: INestApplication;
-  if (bootstrapOptions?.expressApp) {
-    app = await NestFactory.create(AppModule, new ExpressAdapter(bootstrapOptions?.expressApp), nestOptions);
-  } else {
-    app = await NestFactory.create(AppModule, { bufferLogs: true, ...nestOptions });
-  }
```

Comment:
I didn't find any use for this condition to exist, there seem to be no place that would specify `bootstrapOptions?.expressApp` and NestJS uses express by default:

https://docs.nestjs.com/first-steps#platform

> The @nestjs/platform-express package is used by default. Many users are well served with Express, and need take no action to enable it